### DISCUSSION TITLE
node-red non parte

### DISCUSSION THREAD

**Discussion by effe-7** [indomus/forum#2697] [HUB: Home Assistant]
ho installato il componente Node-RED ma quando lo eseguo non parte e ripropone sempre che "il componente non è in esecuzione, vuoi attivarlo?" 
grazie

**Comment by snakuzzo** [comment]
e quindi ?
Uno come può darti una mano in base a quello che hai scritto ?
Sfera di cristallo ?

Posta qualche evidenza (log, configurazione, cosa e come hai installato, etc)

**Comment by effe-7** [comment]
scusa, comunque grazie andando a vedere i log ho capito il problema... non avendo il sistema su https dovevo disabilitare ssl

grazie.

**Reply by snakuzzo** [reply]
Ok
